assets/scss/components/_summary-card.scss

```diff
@@ -12,3 +12,19 @@
     color: govuk-colour("green");
   }
 }
+
+.app-summary-card__change-link {
```

Comment:
Custom CSS to handle the alignment of the "Change" link next to the `<h3>` as per the design